knplabs/knp-snappy-bundle/1.4/etc/packages/knp_snappy.yaml

```diff
@@ -0,0 +1,10 @@
+# Read Documentation: https://github.com/KnpLabs/KnpSnappyBundle/blob/v1.4/README.markdown
+knp_snappy: ~
+#    temporary_folder: %kernel.cache_dir%/snappy
+#    pdf:
+#        enabled:    true
+#        options: []
+#    image:
+#        enabled:    true
+#        options: []
```

Comment:
Well, if 99% of people using the bundle don't need to configure anything, it is indeed best to rely on default values. But then, my point is that this file is useless in 99% of projects, and so does not need to be there at all (and so the recipe is useless after that, as it is exactly the same than the auto-created recipe if your remove this file).